nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -267,4 +269,10 @@ def get_compress_pipeline(
 class OVAWQAlgoAlgoBackend(OVWeightCompressionAlgoBackend):
     @staticmethod
     def get_awq_patterns():
-        return get_awq_patterns(om.OVMatMulMetatype, om.OVMultiplyMetatype)
+        return get_awq_patterns(om.OVMatMulMetatype, om.OVMultiplyMetatype, ATOMIC_ACTIVATIONS_OPERATIONS)
+
+    @staticmethod
+    def scale_insertion_command(source_node, next_nodes, source_node_output_port, scale):
```

Comment:
The update for the `AWQAlgoBackend` class with this method was missed.

Reply:
Done